packages/frontend/pages/orders/checkout.tsx

```diff
@@ -0,0 +1,25 @@
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { wrapper } from 'store';
+import { withPublic } from '@hocs';
+import { Layout } from '@components/layout';
+import { CheckoutPageInner } from '@components/order';
+
+export const getStaticProps = withPublic(
```

Comment:
This page should be protected. On BE we need to get order by orderId + userId from token